app/bundles/PageBundle/Views/Page/list.html.php

```diff
@@ -62,6 +61,37 @@
                     ]
                 );
 
+                echo $view->render(
+                    'MauticCoreBundle:Helper:tableheader.html.php',
+                    [
+                        'sessionVar' => 'page',
+                        'orderBy'    => 'p.dateAdded',
+                        'text'       => 'mautic.lead.import.label.dateAdded',
+                        'class'      => 'col-page-dateAdded visible-md visible-lg',
+                    ]
+                );
+
+                echo $view->render(
+                    'MauticCoreBundle:Helper:tableheader.html.php',
+                    [
+                        'sessionVar' => 'page',
+                        'orderBy'    => 'p.dateModified',
+                        'text'       => 'mautic.lead.import.label.dateModified',
+                        'class'      => 'col-page-dateModified visible-md visible-lg',
+                        'default'    => true,
```

Comment:
I can't set `dir` to desc because if i do it icon will be fixed to DESC, i added another parameter `orderDir`  that will be considered if `default` param is true.
After this fix if you click a second time on modified date the order will be continue to be DESC, this happens setListFilters method on CommonController has ASC as default behavior, it doesn't see orderbydir variable setted in session and convert again to DESC. So i needed to do another fix : set session orderbydir to DESC inside PageController if it's not defined.

Reply:
You're right.  But I think it could be done more simple.

What about just new param defaultDir => 'DESC' what replace 

`$dir          = (!empty($dir)) ? $dir : $app->getSession()->get("mautic.{$sessionVar}.orderbydir", $defaultDir ?? 'ASC');`